Update faq.md (#1717)
status-im/nim-beacon-chain: docs/the_nimbus_book/src/faq.md

> Remember that a validator’s vote is weighted by the amount it has at stake.

## Can I stop my validator for a few days and then start it back up again?

> Yes but, under normal conditions, you will lose an amount of ETH roughly equivalent to the amount of ETH you would have gained in that period. In other words, if you stood to earn ≈0.01 ETH, you would instead be penalised ≈0.01 ETH.

Reviewer: it's good to note here that the decay curve gets steeper as more time passes. there are some sites out there describing the economics of staying offline more thoroughly - might be good to consider showing a basic table?

Author: i think this is only true in the case where > 1/3 are offline (i.e. under abnormal conditions)?

Author: as fas as i understand, if the chain is finalising, the penalties don't increase

## When should I top up my validator's balance?

The answer to this question very much depends on how much ETH you have at your disposal.

> You should certainly top up if your balance is close to 16 ETH: this is to ensure you don't get kicked out of the validator set (which automatically happens if your balance falls below 16 ETH).

Reviewer: ```suggestion
You should certainly top up if your balance is close to 16 ETH: this is to ensure you don't get removed from the validator set (which automatically happens if your balance falls below 16 ETH).
```
